File: docs/fortran/algorithmes/sleeper.f90
!  First programs in Fortran.
!  Julien VALENTIN (2022)  julien.vlnt@gmail.com
!  A program that sleeps for two seconds.

!  TO FIX : how to capture these two seconds ? tac-tic = 0 here.
program main
    real :: tic, tac
    call time(tic)
    call sleep(2)    ! [s]
    call time(tac)
    print '("I slept for ", f6.3, " s.")', tac-tic
end program

File: docs/python3/algorithmes/sleeper.py
#  First programs in Python
#  Julien VALENTIN (July, 2022)  julien.vlnt@gmail.com
#  A program that sleeps for two seconds.
import time

tic = time.time()
time.sleep(2)    # [s]
tac = time.time()

print(f"I slept for {tac-tic} s.")
